velox/connectors/hive/HiveConfig.h

```diff
@@ -178,6 +178,12 @@ class HiveConfig {
   static constexpr const char* kS3UseProxyFromEnv =
       "hive.s3.use-proxy-from-env";
 
+  /// Timestamp unit used during Velox-Arrow conversion.
+  static constexpr const char* kArrowBridgeTimestampUnit =
+      "arrow-bridge-timestamp-unit";
```

Comment:
@rui-mo Can we name prefix it with `hive.parquet.writer.` to be specific and consistent?

Reply:
Updated as suggested. Thanks.